examples/tutorials/Differentiation_Infrastructure_in_Deepchem.ipynb

```diff
@@ -0,0 +1,527 @@
+{
```

Comment:
<p>Explain the types of deep architectures these primitives could be useful in. For example, DFT. I think there are also deep implicit methods that use root finding as part of the architecture. Also some of the neural ODE style papers. Add citations. Be explicit and precise</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3912/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Done